### PR TITLE
Project proposal v3

### PR DESCRIPTION
**Pull request by aldojasb**
The third version of the project proposal. I did some updates in the EDA conclusions. Feel free to change.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,46 @@
-# Group13_NamePending
-Group member: Son Chau
-Nikita
-yair
-Aldo
+# Quality white wine predictor
+Authors: Aldo Saltao Barros, Nikita Shimberg, Yair Guterman, Son Chou
```
Comment (NikitaShymberg) [inline README.md]: Typo: should be Son Chau

File: README.md [hunk 2]
```diff
@@ -1,5 +1,46 @@
...
+
+## About 
+
+**Goal**
+This project aims to determine which features are the best quality white wine indicators and generate insights into each of these factors to our model’s wine quality.
```
Comment (NikitaShymberg) [inline README.md]: I think that last "to our model’s wine quality" can be removed.

Also, is our goal to build a model to predict wine quality given wine features, or is it to determine which features indicate best quality?

File: README.md [hunk 3]
```diff
@@ -1,5 +1,46 @@
...
+12. quality (score between 0 and 10)
+
+
+**How to analyze the data**
+Our task here is to focus on what white wine features are important to get the promising result. For the purpose of classification model and evaluation of the relevant features, we are using algorithms such as 1) Decision Tree, 2) SVC, 3) K-NN, 4) Navie Bayes, and 5) Logistic Regression. We will test them and compare what is the best model and the best parameters to use in this prediction.
+
+
+**EDA first conclusions**
+
+According to our first EDA, we do not have a balanced database, our wines are concentrated around quality 5 and 7.5 (around 80% of data points). Besides, we have a couple of signs about some variables. For instance, it appears that the higher the alcohol level, the better the wine quality. Additionally, the smaller the chlorides and total sulphur dioxide the better the wine quality. Some variables seem do not to influence wine quality, such as free sulphur dioxide, fixed acidity, and sulphates.
```
Comment (NikitaShymberg) [inline README.md]: Re: Some variables seem do not to influence wine quality, such as free sulphur dioxide, fixed acidity, and sulphates.

I would say something like these factors don't influence the wine quality _on their own_. When combining these variables, they might indeed influence wine quality. For example, maybe good quality wine is low in sulphates _and_ low in free sulphur dioxine.

File: README.md [hunk 4]
```diff
@@ -1,5 +1,46 @@
...
+Authors: Aldo Saltao Barros, Nikita Shimberg, Yair Guterman, Son Chou
+
+## About 
+
+**Goal**
+This project aims to determine which features are the best quality white wine indicators and generate insights into each of these factors to our model’s wine quality.
+
+**Introduction**
+According to experts, wine is differentiated according to its smell, flavour, and colour, but most people are not wine experts to say that wine is good or bad. Those variables and others not mentioned so far compose what we define as the quality of the wine. The quality of a wine is important for the consumers as well as the wine industry. For instance, industry players are using product quality certifications to promote their products. However, this is a time-consuming process and requires the assessment given by human experts, which makes this process very expensive. Nowadays, machine learning models are important tools to replace human tasks and, in this case, a good wine quality prediction can be very useful in the certification phase. For example, an automatic predictive system can be integrated into a decision support system, helping the speed and quality of the performance.
```
Comment (NikitaShymberg) [inline README.md]: Re: Those variables and others not mentioned so far compose what we define as the quality of the wine

I would rephrase this to something like "the quality of the wine is determined by many variables including, but not limited to, the ones mentioned previously".

**Review by NikitaShymberg — CHANGES_REQUESTED** [review]
Overall looks very good! I just mentioned some changes, and it might be a good idea to check in about the research question/goal with the whole team.

File: README.md (hunk 1 above)
Comment (aldojasb) [reply README.md]: ok! Fixing that

**Comment by aldojasb** [comment]
I've fixed those errors in the v06.
